idaes/core/surrogate/pysmo_surrogate.py

```diff
@@ -328,6 +328,19 @@ def _create_model(self, pysmo_input, output_label):
     def _get_metrics(self, model):
         return {"RMSE": model.errors["MSE"] ** 0.5, "R2": model.errors["R2"]}
 
+    def get_confidence_intervals(self, model, confidence=0.95):
```

Comment:
You don't indicate the type or structure of the return value in the docstring.

Reply:
Done: type now indicated in docstring.